p2p/gossip.go

```diff
@@ -111,9 +99,7 @@ func (g *Gossiper) ProcessMessages(ctx context.Context) {
 			continue
```

Comment:
We don't need this `GetFrom()` condition in here anymore.

Reply:
Why exactly is that ? Will the node continue to process the it's own messages (not sure whether this is a great way to put it) ? Will it also be handled in the validator or is it not needed at all?